app/src/main/res/layout/addfeed.xml

```diff
@@ -142,6 +142,20 @@
                     android:clickable="true"
                     android:text="@string/browse_gpoddernet_label"/>
 
+            <TextView
+                android:id="@+id/btn_search_podcastindex"
```

Comment:
Please use the same indention as the rest of the file. I know it is pretty inconsistent throughout the project but I think we should at least have it consistent within each file.

Reply:
Intended like the rest of the file.